cmd/vic-machine/common/id.go

```diff
@@ -30,16 +28,12 @@ func (i *VCHID) IDFlags() []cli.Flag {
 		cli.StringFlag{
 			Name:        "id",
 			Value:       "",
-			Usage:       "The ID of the Virtual Container Host - not supported until vic-machine ls is ready",
+			Usage:       "The ID of the Virtual Container Host",
```

Comment:
Is this the full `VirtualMachine:vm-xyz` or just the last portion? If it's just the last portion that that's what we should output in the `ls` output for the ID field.
If this is the full : separated string then I think we should change it to be just `vm-xyz` as it will _always_ be prefixed by VirtualMachine